src/main/java/uk/gov/pay/directdebit/payers/model/BankAccountDetails.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.pay.directdebit.payers.model;
+
+public class BankAccountDetails {
+
+    private final String accountNumber;
```

Comment:
`accountNumber` and `sortCode` are `String`s because we can have some numbers starting with `0`, or?

Reply:
Yes, they are not real numbers, it's just that incidentally they are made of numbers :) they can start with 0 etc. etc.